registry/consul/registry.go

```diff
@@ -95,6 +95,7 @@ func (r *consulRegistry) register(url common.URL) error {
 	return r.client.Agent().ServiceRegister(service)
 }
 
+// Unregister service from consul registry center
```

Comment:
Unregister deletes service from consul registry center

Reply:
I think `Unregister service from consul registry center` is a full sentence.